research/delf/delf/python/training/download_dataset.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
```

Comment:
One small comment for these two files: what do you think about changing the indentation from 4 spaces to 2 spaces? This will make the lines shorter for all cases and make them a bit more readable IMHO.

Reply:
Sure, that makes sense. I changed the indentation to 2 spaces.